pkg/controller/upgradeconfig/cluster_upgrader.go

```diff
@@ -245,6 +245,9 @@ func CommenceUpgrade(c client.Client, m maintenance.Maintenance, upgradeConfig *
 	clusterVersion.Spec.Overrides = []configv1.ComponentOverride{}
 	clusterVersion.Spec.DesiredUpdate = &configv1.Update{Version: upgradeConfig.Spec.Desired.Version}
 	clusterVersion.Spec.Channel = upgradeConfig.Spec.Desired.Channel
+
+	//Record the timestamp when we start the upgrade
+	metrics.UpdateMetricUpgradeStartTime(time.Now(), upgradeConfig.Name)
```

Comment:
In the maintenance client, we explicitly get the time as UTC when dealing with it. Thoughts on maybe doing the same here so that we're consistent across all the code? eg.

```suggestion
	metrics.UpdateMetricUpgradeStartTime(time.Now().UTC(), upgradeConfig.Name)
```